utils/install_v2.sh

```diff
@@ -174,7 +194,11 @@ check_os_arch() {
 					;;
 			esac
 			if [ "${LEGACY}" == 1 ]; then
-				RELEASE_PKG="manylinux2014_${ARCH}.tar.gz"
+				if version_ge "${VERSION}" "0.15.0"; then
+					RELEASE_PKG="manylinux_2_28_${ARCH}.tar.gz"
+				else
+					RELEASE_PKG="manylinux2014_${ARCH}.tar.gz"
+				fi
```

Comment:
Hi @SAtacker 
From 0.15.0, the manylinux assets is changed from `2014` to `_2_28`.